tests/test_core/test_system_core.py

```diff
@@ -3,12 +3,19 @@
 import os
 sys.path.append(os.path.join(os.path.dirname(__file__), "..", ".."))
 
+# logging
```

Comment:
Because it's amazing how often I end up having to add debugging in unit tests to work out what is going wrong...